### DISCUSSION TITLE
Rad Pro: Logo and icons

### DISCUSSION THREAD

**Discussion by ihrapsa** [Gissio/radpro#86] [Show and tell]
Hello,
I thought this project deserves a a nice logo so I made these:

![image](https://github.com/Gissio/radpro/assets/40600040/a5d9091c-0418-499c-992c-587e2451f85b) 

I was also thinking about creating  a menu items icon set for both TFT LCD and graphic LCD screens.

What do you think? 😁

**Comment by ihrapsa** [comment]
I personally like number 1 and 3 most

**Comment by jonwhite227** [comment]
Nice! 4 & 5 are my favourites.
